src/config.h

```diff
@@ -158,7 +160,7 @@ typedef struct {
   // Control user data obfuscation in logs
   bool hideUserDataFromLog;
   // Stretch factor for BM25STD.TANH scoring function
-  uint64_t BM25STD_TanhStretchFactor;
+  uint64_t BM25STD_TanhFactor;
```

Comment:
redundant and duplicating here (we have a `requestConfigParams` section)

Reply:
Right forgot to clean that